pack/core/core_2026.json

```
@@ -0,0 +1,211 @@
+[
+    {
+        "code": "12016",
+        "cost": 3,
+        "deck_limit": 2,
+        "faction_code": "guardian",
+        "flavor": "\"I'm not here to help you shop; I'm here to keep you alive.\"",
+        "health": 2,
+        "illustrator": "Billy Norrby",
+        "name": "Bodyguard",
+        "pack_code": "core_2026",
+        "position": 16,
+        "quantity": 2,
+        "sanity": 1,
+        "skill_combat": 1,
+        "slot": "Ally",
+        "text": "Bodyguard may be assigned damage dealt to other investigators at your location.\n[reaction] When Bodyguard is defeated: Deal 1 damage to an enemy at your location.",
+        "traits": "Ally.",
+        "type_code": "asset",
+        "xp": 0
+    },
+    {
+        "code": "12021",
+        "cost": 3,
+        "deck_limit": 2,
+        "faction_code": "guardian",
+        "flavor": "Nothing's gonna keep you down.",
+        "health": 2,
+        "illustrator": "Rafal Hrynkiewicz",
+        "name": "Resilience",
+        "pack_code": "core_2026",
+        "position": 21,
+        "quantity": 2,
+        "sanity": 2,
+        "skill_willpower": 1,
+        "text": "",
+        "traits": "Talent.",
+        "type_code": "asset",
+        "xp": 0
+    },
+    {
+        "code": "12033",
+        "cost": 3,
+        "deck_limit": 2,
+        "faction_code": "seeker",
+        "flavor": "\"I could swear we've been here before...\"",
+        "illustrator": "Pixoloid Studios",
+        "name": "Local Map",
+        "pack_code": "core_2026",
+        "position": 33,
+        "quantity": 2,
+        "skill_agility": 1,
+        "slot": "Hand",
+        "text": "Uses (4 secrets).\n[action] Spend 1 secret: <b>Investigate ([intellect]).</b> Investigate a revealed connecting location. You get +1 [intellect] for this investigation. If you succeed, you may exhaust Local Map to move to that location.",
```

The brackets aren't bold in the preview images.
```suggestion
        "text": "Uses (4 secrets).\n[action] Spend 1 secret: <b>Investigate</b> ([intellect]). Investigate a revealed connecting location. You get +1 [intellect] for this investigation. If you succeed, you may exhaust Local Map to move to that location.",
```